package.json

```diff
@@ -28,7 +28,9 @@
   "license": "MIT",
   "devDependencies": {
     "@bpmn-io/bpmnlint-generate-docs-images": "^0.2.0",
+    "bpmn-moddle": "^7.0.4",
     "bpmnlint": "^7.2.1",
+    "camunda-bpmn-moddle": "^5.1.2",
```

Comment:
What is generally interesting about this one is the fact that the new rule will ever only work if the `moddle` has been loaded with it. I.e. on the CLI, which right now does not have a way to specify the moddle packages being used, `implementation` won't ever be properly validated.

Is that an issue? Not sure.

Reply:
That's true. `bpmnlint` as a CLI should somehow pick up the additional schemas, but it's not implemented yet.

Still, this is probably outside of the scope of this issue. The rule will work correctly in any environment where `camunda-bpmn-moddle` is used and this is the only environment where the rule is needed.

I'll have an issue for the `bpmnlint` repo.

Reply:
https://github.com/bpmn-io/bpmnlint/issues/62

Reply:
Thanks for opening that issue. I'm fine to proceed with this PR. We can improve via https://github.com/bpmn-io/bpmnlint/issues/62 as we find time.